src/app/IntervalTimer/interval-timer.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Input, OnDestroy, OnInit, Output, EventEmitter } from '@angular/core';
+
+
+@Component({
+  selector: 'interval-timer',
+  templateUrl: './interval-timer.component.html',
+  styleUrls: ['./interval-timer.component.scss']
+})
+export class IntervalTimer implements OnInit, OnDestroy {
+
+  // Output to emit an Event everytime the Timer reaches 0
+  @Output()
+  intervalTimerEvent: EventEmitter<any> = new EventEmitter()
```

Comment:
```suggestion
  intervalTimerEvent: EventEmitter<void> = new EventEmitter()
```

If you use `void` as type, you do not have to provide a value IIRC.

Regarding the name: I would probably use something like `fire`, `elapse`. Maybe you can think of something better. I am alsways trying to use the native DOM event names as a guideline (e.g. `click`, `keyup`, `focus`, ...)